metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/database/JdbcOperation.java

```diff
@@ -0,0 +1,115 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.database;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.SQLXML;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JdbcOperation {
+
+    private static final Logger logger = LoggerFactory.getLogger(JdbcOperation.class);
+
+    private JdbcConnection jdbcConnection;
+
+    public JdbcOperation(JdbcConnection jdbcConnection) {
+        this.jdbcConnection = jdbcConnection;
+    }
+
+    private String createCompleteUrl(JdbcConnection jdbcConnection) {
```

Comment:
Since we are already using Spring, why not letting it deal with the database connection? This kind of _ad hoc_ JDBC code always fails to support some corner case.

Reply:
To my knowledge JDBC was suggested by ENA team. I can use JDBCTemplate method

Reply:
Just to save you some time...

https://github.com/EbiEga/ega-production/blob/master/file-re-encrypt/src/main/java/uk/ac/ebi/ega/file/re/encrypt/DatabaseConfiguration.java

That is how you configure to read the connection details from properties and create an appropriate DataSource, TransactionManager, JDBCTemplate and if there is more than one connection in the application  also configure properly the HikariCP.

Reply:
done